filters.js

```diff
@@ -199,6 +199,10 @@ function isValidDateString(dateString) {
   return typeof dateString === "string" && moment(dateString, "YYYY-MM-DDTHH:mm:ss.SSSZ", true).isValid();
 }
 
+function isValidNumber(argument) {
+  return argument !== null && argument !== undefined
```

Comment:
We should be checking for NaN as well? This does not guarantee a number, just an object which is not undefined.